tests/monitoring/test_mean_aggregator.py

```diff
@@ -0,0 +1,40 @@
+import theano
```

Comment:
Yes, it is perfectly fine that a file contains multiple tests. The structure of `tests` directory roughly follows the one of `blocks` and the tests are typically located in a file `tests/path/test_file.py` for `blocks/path/file.py`.